Formula/pyenv.rb

```diff
@@ -57,14 +57,18 @@ def install
     chmod "+x", foo_script
 
     # Test versions.
-    versions = shell_output("eval \"$(#{bin}/pyenv init -)\" && pyenv versions").split("\n")
+    versions = shell_output("eval \"$(#{bin}/pyenv init --path)\" " \
```

Comment:
note `pyenv init --path` (which adds the shim dir to `$PATH`) was split from `pyenv init -`. see the [release notes](https://github.com/pyenv/pyenv/releases/tag/v2.0.0)